MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1342,6 +1342,10 @@ private void SetAudioInfoFromTags(MediaInfo audio, Dictionary<string, string> ta
             mb = GetMultipleMusicBrainzId(tags.GetValueOrDefault("MusicBrainz Release Track Id"))
                  ?? GetMultipleMusicBrainzId(tags.GetValueOrDefault("MUSICBRAINZ_RELEASETRACKID"));
             audio.TrySetProviderId(MetadataProvider.MusicBrainzTrack, mb);
+
+            mb = GetMultipleMusicBrainzId(tags.GetValueOrDefault("MusicBrainz Recording Id"))
```

Comment:
But, we don't add more functions to the ProbeResultNormalizer for music. This one is considered legacy and only being used for fallback for a minimal set of tags. The changes to this file should be removed.

Reply:
Done!